tasks/main.yml

```diff
@@ -45,23 +45,25 @@
     command: "{{ maintain_command }} packages lock"
     when: maintain_command is defined
   when: install_receptor_from_rpm|bool
-- name: Extract account information from Satellite and write configs
+- name: Retrieve organization details
   block:
-  - name: List all organizations in Satellite
-    uri:
-      url: "{{ satellite_url }}/katello/api/organizations"
-      user: "{{ satellite_user }}"
-      password: "{{ satellite_password }}"
-      return_content: yes
-      validate_certs: "{{ satellite_validate_certs }}"
-      force_basic_auth: yes
-    register: sat_orgs_json
-  - name: Extract organization configuration per account and save to disk
-    include_tasks: extract_orgs.yml
-    loop: "{{ sat_orgs_json.json | json_query('results[*].id') }}"
-    when: not org_id in skip_satellite_org_id_list
-    loop_control:
-      loop_var: org_id
+    - name: Dump organization details
+      command: foreman-rake katello:receptor:extract_orgs OUTPUT_FILE=/tmp/receptor.json
```

Comment:
what happens if this is ran in parallel for two orgs? should we generate proper tempfile somehow to be on the safe side?

Reply:
The rake task gives details for all organizations so it is not really being run for an organization. The issue with running this multiple times at once is real though.